repo/packages/M/marathon/0/marathon.json

```diff
@@ -1,21 +1,15 @@
 {
-  "id": "{{marathon/framework-name}}",
-  "cpus": {{marathon/cpus}},
-  "mem": {{marathon/mem}},
-  "instances": {{marathon/instances}},
+  "id": "{{marathon.framework-name}}",
+  "cpus": {{marathon.cpus}},
+  "mem": {{marathon.mem}},
+  "instances": {{marathon.instances}},
   "constraints": [["hostname", "UNIQUE"]],
   "ports": [
-    {{marathon/http-port}}
-
-    {{#marathon/https-port}}
-    , {{marathon/https-port}}
-    {{/marathon/https-port}}
+    {{marathon.http-port}}
+    , {{marathon.https-port}}
```

Comment:
If we swap the order of the libprocess port (always required) and the https port (only required if that protocol is enabled) then we can optionally avoid consuming a port we don't intend to bind here.